Projects/ChangeTier/Sources/Views/ChangeTierLandingScreen.swift

```diff
@@ -0,0 +1,265 @@
+import SwiftUI
+import hCore
+import hCoreUI
+import hGraphQL
+
+struct ChangeTierLandingScreen: View {
+    @ObservedObject var vm: ChangeTierViewModel
+    @EnvironmentObject var changeTierNavigationVm: ChangeTierNavigationViewModel
+    @EnvironmentObject var router: Router
+    var contractId: String
```

Comment:
This should be already set in the ChangeTierViewModel as an let property so we don't move it around from the view  to the viewmodel